app/src/main/java/com/microsoft/device/samples/dualscreenexperience/presentation/catalog/ui/CatalogScreen.kt

```diff
@@ -64,6 +81,10 @@ fun Catalog(
     PageViews(pagerState, pages, showTwoPages)
 }
 
+fun lerp(start: Float, stop: Float, fraction: Float): Float {
```

Comment:
what does `lerp` mean? maybe we can use a more meaninful name for this function?